recipes/ipython-sql/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.3.6" %}
+{% set md5 = "d4feb00ac5806d7640b2545a43974766" %}
+
+package:
+  name: ipython-sql
+  version: {{ version }}
+
+source:
+  fn: ipython-sql-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipython-sql/ipython-sql-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [py2k]
```

Comment:
We should be able to drop this `skip` line as `conda-build` version `1.21.11` is out and shouldn't run into the same issues.